allure-testng-adaptor/src/main/java/ru/yandex/qatools/allure/testng/AllureTestListener.java

```diff
@@ -2,13 +2,13 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.testng.IConfigurationListener;
+import org.testng.ISuite;
+import org.testng.ISuiteListener;
 import org.testng.ITestContext;
-import org.testng.ITestListener;
 import org.testng.ITestNGMethod;
 import org.testng.ITestResult;
-import org.testng.SkipException;
-import org.testng.TestException;
+import org.testng.internal.IResultListener;
```

Comment:
is it ok to use `internal` classes? I guess `internal` means that it is not a part of the public API and it can be changed in any time